build_tools/github_actions/configure_ci.py

```diff
@@ -95,7 +95,6 @@ def contains(cls, val):
 SKIP_PATH_PATTERNS = [
     "docs/*",
     "third_party/mkdocs-material/*",
-    "experimental/*",
```

Comment:
We should probably promote pkgci out of experimental at this point. Or at least move the config files out of there